Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.4
```

Comment:
3.7 is the latest release.

Reply:
Ack. I will change this. We should then also align other projects.

Reply:
I use this whenever creating new images. But you're right we should update existing images too.